### ISSUE TITLE
Google forms linear scale has a max value of 10

### ISSUE THREAD

**Issue by 16OscarClarke** [16OscarClarke/2.4_2.8DITWebsiteInternal#8]
this means that if the user requests a larger number then they are unable to do so
